### DISCUSSION TITLE
disable property not working

### DISCUSSION THREAD

**Discussion by bishrkhouli1** [primefaces/community#155] [PrimeNG]
i am binding [ disabled ] property to a function that function return true or false  

the input is a part of array of checkboxes  when i push new element to the array  the disabled not work 

i tried the same logic on  native checkbox and it worked as expected 

what can be the problem 

i am  using angular 11 

"primeng": "11.4.5"

![image](https://github.com/primefaces/community/assets/93731686/47e1b16f-26ee-459d-ab82-bda74767350c)

![image](https://github.com/primefaces/community/assets/93731686/71c23fe4-7946-419f-a1b3-95669e6f9eb5)

**Comment by cetincakiroglu** [comment]
Hi,

Could you please create a stackblitz example so we can identify the issue?
